puller_state

```diff
@@ -1 +1 @@
-490692396
\ No newline at end of file
+490692413
```

Comment:
I thought we agreed to delete this file.

Reply:
I put this file to gitignore.